spec/reek/cli/application_spec.rb

```diff
@@ -44,6 +44,18 @@
                configuration: Reek::Configuration::AppConfiguration,
                options: Reek::CLI::Options)
       end
+
+      context 'when a stdin filename is provided' do
+        let(:app) { described_class.new ['--stdin-filename', 'foo.rb'] }
+
+        it 'assumes that filename' do
```

Comment:
I don't see the specific filename being checked in the implementation of this spec?

Reply:
I should probably just remove this spec. I wanted somewhere in the regular specs (not just features) that would test the `--stdin-filename` option, and this seemed like the most logical place to do so. However, once I dug in, everything in here is so mocked out I wasn't sure how to actually test it, or even what I should be testing. I'm also surprised it passed, had it failed I probably would have just removed it.